ConsoleApp1/Snake.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Timers;
+
+namespace ConsoleApp1
+{
+    public class Snake
+    {
+        public static int score { get; private set; } = 0;
+        public char SnakeChar { get; set; }
+        public static Point point { get; private set; }
+        public static List<Point> Body { get; private set; }
+
+        public Snake()
+        {
+            point = new Point(Board.Size / 2, Board.Size / 2);
+            Body = new List<Point>
+            {
+                new Point(Board.Size / 2, Board.Size / 2),
+                new Point((Board.Size / 2), Board.Size / 2),
+                new Point((Board.Size / 2), Board.Size / 2),
+            };
+        }
+
+        public void MoveSnake(object? sender, ElapsedEventArgs e)
+        {
+            Direction.GetDirection();
+            int side = (int)Direction.eDirection;            
+            switch (side)
+            {
+                case 0: Body.Add(new Point(Body.Last().x, Body.Last().y - 1)); SnakeChar = '^'; break;
```

Comment:
`Body.Last()` is calculated every time you called it, so move it to variable and reuse it